### PR TITLE
Continuous zooming in separate frames mode

### LINKED ISSUE #5049: Classifier continuous zoom in/out broken in separate frames mode
## Package
lib-classifier

## Describe the bug
In separate frames mode, the +/- buttons next to each frame do not continuously zoom in or out while pressed. Instead, they zoom once when the pointer is released ie. on click.

Pressing each button with Enter does continuously zoom in and out.

## To Reproduce
1. Go to https://www.zooniverse.org/projects/fulsdavid/the-daily-minor-planet/classify/workflow/22354
2. Switch the subject to 'separate frames view'.
3. Press and hold the +/- buttons with a pointer device (mouse, trackpad, touchscreen etc.)

## Expected behavior
The subject should smoothly zoom in and out while each button is pressed.

### PR DESCRIPTION
**Pull request by eatyourgreens**
Separate frames mode uses its own callback functions for zoom, so they need to be called continuously while the zoom buttons are pressed.

I've also updated the tests to expect each zoom callback to run once on button click (see #5043.)

_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

## Package
lib-classifier

## Linked Issue and/or Talk Post
- closes #5049.

## How to Review
You can test this on Daily Minor Planet in the standalone classifier:
https://localhost:8080/?project=fulsdavid/the-daily-minor-planet&env=production

Pressing the +/- buttons next to each frame, with either a pointing device or the keyboard, should continuously zoom while each button is pressed.

- [x] tests are passing.
- [x] the subject zooms continuously when the buttons are pressed with pointerdown.
- [x] the subject zooms continuously when the buttons are pressed with the keyboard.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Bug Fix
- [ ] The PR creator has listed user actions to use when testing if bug is fixed
- [ ] The bug is fixed
- [ ] Unit tests are added or updated

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ function ZoomInButtonContainer({ separateFrameZoomIn = () => true }) {
   return (
     <ZoomInButton
       disabled={disabled}
-      onClick={separateFramesView ? separateFrameZoomIn : zoomIn}
+      onClick={zoomCallback}
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js:46]: This fires continuously while the Enter key is held down, but I can't see why.

Comment (eatyourgreens) [reply packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js]: Test pen here: https://codepen.io/eatyourgreens/pen/abQaNqp

This seems to be the way that buttons work by default, at least in MacOS.

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js [hunk 2]
```diff
@@ -21,29 +21,33 @@ function storeMapper(classifierStore) {
   }
 }
 
-function ZoomInButtonContainer({ separateFrameZoomIn = () => true }) {
-  const { disabled, separateFramesView, zoomIn } = useStores(storeMapper)
```
Comment (goplayoutside3) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js:25]: Removing `separateFramesView` here means it should also be removed from the `storeMapper` function in ZoomInButtonContainer and ZoomOutButton Container.

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomOutButton/ZoomOutButtonContainer.js [hunk 3]
```diff
@@ -21,29 +21,33 @@ function storeMapper(classifierStore) {
   }
 }
 
-function ZoomOutButtonContainer({ separateFrameZoomOut = () => true }) {
-  const { disabled, separateFramesView, zoomOut } = useStores(storeMapper)
```
Comment (goplayoutside3) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomOutButton/ZoomOutButtonContainer.js:25]: Remove `separateFramesView` from the `storeMapper` in this file too.

**Review by goplayoutside3 — APPROVED** [review]

**Pull request merged**